hummingbot/client/command/start_command.py

```diff
@@ -76,6 +78,21 @@ async def start_check(self,  # type: HummingbotApplication
         self._notify(f"\nStatus check complete. Starting '{self.strategy_name}' strategy...")
         if global_config_map.get("paper_trade_enabled").value:
             self._notify("\nPaper Trading ON: All orders are simulated, and no real orders are placed.")
+
+        connector = get_strategy_config_map(self.strategy_name).get("exchange").value
```

Comment:
This will only work for PMM strategy, as other strategies might not have connector config key as "exchange". A better way to get configured exchanges is to use `required_exchanges` from `setting`, this is a list so you'll have to loop through them.

Reply:
Thanks @Nullably for the catch. I pushed a new commit, tested on cross exchange and arbitrage strategy.

![image](https://user-images.githubusercontent.com/50150287/98101146-c445c300-1ecc-11eb-8df5-4dbc2d823414.png)